src/apps/external_executor/protobuf/build.sh

```diff
@@ -2,28 +2,27 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the Apache 2.0 License.
 
+if [ "$#" -ne 2 ]; then
+    echo "Usage: $0 <path_to_proto_file> <output_directory>"
+fi
+
 THIS_DIR=$( dirname "${BASH_SOURCE[0]}" )
+SOURCE_FILE=${1}
+GENERATED_DIR=${2}
 
 if [ ! -f "env/bin/activate" ]
     then
         python3.8 -m venv env
 fi
 
 source env/bin/activate
-pip install -U -r "${THIS_DIR}/requirements.txt"
+pip install -U -r "${THIS_DIR}/requirements.txt" > /dev/null
```

Comment:
```suggestion
pip install -q -U -r "${THIS_DIR}/requirements.txt"
```

I think this is similar but safer.